patterns/hidden-404.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Title: 404
+ * Slug: twentytwentyfive/hidden-404
+ * Inserter: no
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+
+<!-- wp:group {"align":"wide","layout":{"type":"constrained","justifyContent":"left"}} -->
+<div class="wp-block-group alignwide">
+	<!-- wp:columns {"align":"wide","style":{"spacing":{"padding":{"top":"var:preset|spacing|50","bottom":"var:preset|spacing|50"},"blockGap":{"left":"var:preset|spacing|60"}}}} -->
+	<div class="wp-block-columns alignwide" style="padding-top:var(--wp--preset--spacing--50);padding-bottom:var(--wp--preset--spacing--50)">
+		<!-- wp:column {"width":"40%"} -->
+		<div class="wp-block-column" style="flex-basis:40%">
+			<!-- wp:image {"sizeSlug":"large","linkDestination":"none","className":"is-resized"} -->
```

Comment:
```suggestion
			<!-- wp:image {"sizeSlug":"large"} -->
```

Image is not resized and linked.

Reply:
These are the default values when you add an image in the block editor.
If they are removed, the editor has to manually add the class name and change the markup to this:
<!-- wp:image {"id":19,"className":"size-large"} -->
Please motivate why it should be removed?